v4/export/consistency_test.go

```diff
@@ -5,63 +5,55 @@ package export
 import (
 	"context"
 	"errors"
-	"strings"
+	"testing"
 
 	tcontext "github.com/pingcap/dumpling/v4/context"
+	"github.com/stretchr/testify/require"
 
 	"github.com/DATA-DOG/go-sqlmock"
 	"github.com/go-sql-driver/mysql"
-	. "github.com/pingcap/check"
 )
 
-var _ = Suite(&testConsistencySuite{})
+func TestConsistencyController(t *testing.T) {
+	t.Parallel()
 
-type testConsistencySuite struct{}
-
-func (s *testConsistencySuite) assertNil(err error, c *C) {
-	if err != nil {
-		c.Fatal(err.Error())
-	}
-}
-
-func (s *testConsistencySuite) assertLifetimeErrNil(tctx *tcontext.Context, ctrl ConsistencyController, c *C) {
-	s.assertNil(ctrl.Setup(tctx), c)
-	s.assertNil(ctrl.TearDown(tctx), c)
-}
-
-func (s *testConsistencySuite) TestConsistencyController(c *C) {
 	db, mock, err := sqlmock.New()
-	c.Assert(err, IsNil)
-	defer db.Close()
+	require.NoError(t, err)
+	defer func() {
+		_ = db.Close()
```

Comment:
why some are converted to `require.NoError(t, x.Close())` while these are left as `_ = x.Close()`

Reply:
Because several cases fail with `require.NoError(t, x.Close())`. The original code doesn't check the err but it is a nit win we check it if tests are well written. For those failed, I don't have time to investigate deeply.